test/jb_projects/challenge_handle.test.js

```diff
@@ -0,0 +1,164 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { getTimestamp } from '../helpers/utils';
+
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+
+import jbOperatoreStore from '../../artifacts/contracts/JBOperatorStore.sol/JBOperatorStore.json';
+
+describe('JBProjects::challengeHandle(...)', function () {
+  const PROJECT_HANDLE = 'PROJECT_1';
+  const PROJECT_HANDLE_NOT_TAKEN = 'PROJECT_2';
+  const PROJECT_HANDLE_NOT_TAKEN_2 = 'PROJECT_3';
+  const METADATA_CID = '';
+  const PROJECT_ID = 1;
+
+  async function setup() {
+    let [deployer, projectOwner, ...addrs] = await ethers.getSigners();
+
+    let mockJbOperatorStore = await deployMockContract(deployer, jbOperatoreStore.abi);
+
+    let jbProjectsFactory = await ethers.getContractFactory('JBProjects');
+    let jbProjectsStore = await jbProjectsFactory.deploy(mockJbOperatorStore.address);
+
+    return {
+      projectOwner,
+      deployer,
+      addrs,
+      jbProjectsStore,
+    };
+  }
+
+  it("Doesn't challenge if inexistent projectId", async function () {
+    const { projectOwner, deployer, jbProjectsStore } = await setup();
+
+    await jbProjectsStore
+      .connect(deployer)
+      .createFor(
+        /*owner=*/ projectOwner.address,
+        /*handle=*/ ethers.utils.formatBytes32String(PROJECT_HANDLE),
+        /*METADATA_CID=*/ METADATA_CID,
+      );
+
+    await jbProjectsStore
+      .connect(projectOwner)
+      .transferHandleOf(
+        /*projectId=*/ PROJECT_ID,
+        /*address=*/ deployer.address,
+        /*handle=*/ ethers.utils.formatBytes32String(PROJECT_HANDLE_NOT_TAKEN),
+      );
+
```

Comment:
Most of this setup seems irrelevant to this test? I would think we just need to challenge a unclaimed handle, no?